.github/workflows/test-and-deploy.yml

```diff
@@ -0,0 +1,73 @@
+name: Test and Deploy
+on:
+  push:
+    branches: [ '*' ]
+    tags: [ '*' ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    # Run automatically at 8AM PST Monday-Friday
+    - cron: '0 15 * * 1-5'
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    strategy:
+      matrix:
+        python-version: [ '2.7', '3.5', '3.6', '3.7', '3.8', '3.9' ]
+    steps:
+      - name: Checkout sendgrid-python
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
Don't think we need to do a deep fetch here

Reply:
Removed the argument and let it default to fetching the triggering commit only.